swerve100/src/main/java/frc/robot/subsystems/SwerveModuleFactory.java

```diff
@@ -147,9 +147,9 @@ public static SwerveModule AMModule(
 
         // Drive(IVE FF
         SimpleMotorFeedforward driveFeedforward = new SimpleMotorFeedforward(//
-                0.04, // kS
-                0.21, // kV
-                0.01); // kA
+                0.045, // kS
+                0.15, // kV
```

Comment:
could you say more about the kV value here?  does the FF do the right thing when the speed is constant?